packages/lightning-lsp-common/src/resources/sfdx/typings/lds.d.ts

```diff
@@ -1163,14 +1448,70 @@ declare module 'lightning/analyticsWaveApi' {
      *
      * https://developer.salesforce.com/docs/component-library/documentation/en/lwc/reference_wire_adapters_get_analytics_limits
      *
-     * @param licenseType The response includes dataflow jobs with this license type. Valid values are `EinsteinAnalytics` or `Sonic`.
+     * @param licenseType The Tableau CRM license types. Valid values are `EinsteinAnalytics` or `Sonic`.
      * @param types The types of limits used in Tableau CRM.
      *              Valid values are `BatchTransformationHours`, `DatasetQueries`, `DatasetRowCount`,
      *              `MaxDailyRowsHighOutputCon`, `MaxDailyRowsLowOutputCon`, `MaxDailyRowsMedOutputCon`,
      *              `MaxDailySizeHighOutputCon`, `MaxDailySizeLowOutputCon`, `MaxDailySizeMedOutputCon`,
      *              or `OutputLocalConnectorVolume`.
      */
-    export function getAnalyticsLimits(licenseType: string, types?: string[]): void;
+    export function getAnalyticsLimits(licenseType?: string, types?: string[]): void;
+
+    /**
+     * Wire adapter to retrieve the Connector for Tableau CRM.
+     *
+     * https://developer.salesforce.com/docs/component-library/documentation/en/lwc/reference_wire_adapters_get_data_connector
+     *
+     * @param connectorIdOrApiName The ID of the connector.
+     */
+    export function getDataConnector(connectorIdOrApiName: string): void;
+
+    /**
+     * Wire adapter to retrieve the collection of Connectors for Tableau CRM.
+     *
+     * https://developer.salesforce.com/docs/component-library/documentation/en/lwc/reference_wire_adapters_get_data_connectors
+     *
+     * @param category The categories that the data connector belongs to. Valid values are:
+     *                 AdvancedPropertiesSupport, BatchRead, Direct, FileBased, FilterSupport, MuleSoft, Output
+     * @param connectorType The type of Tableau CRM connector.
+     * @param scope The type of scope to be applied to the returned collection. Valid values are:
+     *              Created​By​Me, Mru (Most Recently Used), Shared​With​Me
+     */
+    export function getDataConnectors(category?: string, connectorType?: string, scope?: string): void;
+
+    /**
+     * Wire adapter to retrieve the collection of source fields for a particular source object.
+     *
```

Comment:
https://developer.salesforce.com/docs/component-library/documentation/en/lwc/reference_wire_adapters_get_data_connector_source_fields
(I missed this in the list I gave you)